src/core/Button/style.ts

```diff
@@ -8,12 +8,16 @@ import {
   getSpaces,
 } from "../styles";
 
-const sdsPropNames = ["isAllCaps", "isRounded", "sdsStyle", "sdsType"];
+const sdsPropNames = [
+  "isAllCaps",
+  "isRounded",
+  "sdsStyle",
+  "sdsType",
+  "sdsStage",
```

Comment:
```ts
export interface SdsProps {
  isAllCaps?: boolean;
  isRounded?: boolean;
  sdsStyle?: "minimal" | "rounded" | "square";
  sdsType?: "primary" | "secondary";
}
```

ah since `Button` component's interface doesn't include `sdsStage`, it feels like this doesn't belong here?

Reply:
Oops, well spotted :D 
Fixed 👍🏻